README.md

```diff
@@ -1003,6 +1003,53 @@ Vue Storefront is a Community effort brought to You by our great Core Team and s
             height="40"
           >
         </a>
+</td>
+    <td align="center" valign="middle">
+    <a href="https://www.mediawave.de">
+          <img
+            src="https://blog.vuestorefront.io/wp-content/uploads/2020/05/logo_mediawave.png"
+            alt="Mediawave"
+            height="40"
+          >
+        </a>
+</td>
+          <td align="center" valign="middle">
+<a href="https://www.zaproo.com/">
+          <img
+            src="https://blog.vuestorefront.io/wp-content/uploads/2020/05/zaproo_logo.png"
+            alt="Zaproo"
+            height="40"
+          >
+        </a>
+</td> 
+</tr>
+              <tr>
+      <td align="center" valign="middle">
+<a href="http://www.mcfadyen.com/">
+          <img
+            src="https://blog.vuestorefront.io/wp-content/uploads/2020/05/McFadyenLogo-NewTag.png"
+            alt="McFadyen"
+            height="40"
+          >
+        </a>
+</td>
+    <td align="center" valign="middle">
+    <a href="">
```

Comment:
this should be empty?

Reply:
@tom-aniol please check that, and change target branch to `hotfix/v12.1.3`